askama_derive/src/generator.rs

```diff
@@ -1892,7 +1899,7 @@ fn is_copyable_within_op(expr: &Expr<'_>, within_op: bool) -> bool {
 pub(crate) fn is_attr_self(expr: &Expr<'_>) -> bool {
     match expr {
         Expr::Attr(obj, _) if matches!(obj.as_ref(), Expr::Var("self")) => true,
-        Expr::Attr(obj, _) if matches!(obj.as_ref(), Expr::Attr(..)) => is_attr_self(expr),
+        Expr::Attr(obj, _) if matches!(obj.as_ref(), Expr::Attr(..)) => is_attr_self(obj),
```

Comment:
This seems unrelated? I'd like to have this in a separate commit with a test for it.

Reply:
I've already added the test as suggested by @GuillaumeGomez.

Reply:
I've splitted & squashed the commits.